src/test/java/com/training/basetest/ApiBaseTest.java

```diff
@@ -0,0 +1,31 @@
+package com.training.basetest;
+
+import com.training.utilities.JsonReaderUtility;
+import io.restassured.RestAssured;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ApiBaseTest {
+    protected Map<String, String> map = new JsonReaderUtility().getMap();
+    protected Map<String, Object> headerMap;
+    protected io.restassured.response.Response response;
+    SoftAssertions softAssertions;
+
+    @BeforeEach
+    public void setup(){
+        RestAssured.baseURI = map.get("gaBaseUrl");
+        headerMap = new HashMap();
+        headerMap.put(map.get("gaValidationAppKeyHeaderName"),map.get("gaValidationAppKeyHeaderValue"));
+        headerMap.put(map.get("gaValidationContentTypeHeaderName") ,map.get("gaValidationContentTypeHeaderValue"));
```

Comment:
All this should not be in your base test, move it back to your test class. We should only have the code which would be used by all your tests.
Also lets extend CatApiTest to ApiBaseTest.